### DISCUSSION TITLE
How to set up the project in the vs code

### DISCUSSION THREAD

**Discussion by rj-sh17** [GoogleCloudPlatform/gcsfuse#2107] [General]
Could you please let us know the steps to run the project in our local machine for window environment. 

Further, I am using window 64bit.

Furthermore, we already read your provided documentation(https://docs.trudesk.io/v1.2/) but not able to run the project in my local machine.

**Comment by gargnitingoogle** [comment]
> Could you please let us know the steps to run the project in our local machine for window environment.

@rj-sh17 gcsfuse is not supported for Windows OS. The full list of OS names and version supported is [here](https://cloud.google.com/storage/docs/gcs-fuse?hl=en#frameworks-os-architectures) . 

> we already read your provided documentation(https://docs.trudesk.io/v1.2/) but not able to run the project in my local machine

I am not sure I understand the context. This page has not been written by gcsfuse team.
